cmds/boot/systemboot/main.go

```diff
@@ -18,6 +19,100 @@ import (
 	"github.com/u-root/u-root/pkg/smbios"
 )
 
+const (
+	_FB_OEM_IPMI_NET_FUNCTION = 0x36
+
+	_FB_OEM_SET_PROC_INFO = 0x10
+	_FB_OEM_SET_DIMM_INFO = 0x12
```

Comment:
Aren't these IPMI constants? Isn't it better to move them to the IPMI package?

Reply:
Move to oem.go.